tests/trainers/test_gpt3_text-generation_chinese-base_train.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+import os
+import shutil
+import tempfile
+import unittest
+
+import torch
+
+from modelscope.metainfo import Trainers
+from modelscope.msdatasets import MsDataset
+from modelscope.pipelines import pipeline
+from modelscope.trainers import build_trainer
+from modelscope.utils.constant import Tasks
+from modelscope.utils.hub import Config, read_config, snapshot_download
+from modelscope.utils.test_utils import DistributedTestCase, test_level
+
+
+@unittest.skipIf(not torch.cuda.is_available()
+                 or torch.cuda.device_count() <= 1, 'distributed unittest')
+class TestGPT3BaseTrain(DistributedTestCase):
+
+    def setUp(self):
+        print(('Testing %s.%s' % (type(self).__name__, self._testMethodName)))
+
+        self.tmp_dir = tempfile.TemporaryDirectory().name
+        if not os.path.exists(self.tmp_dir):
+            os.makedirs(self.tmp_dir)
+
+        self.model_dir = snapshot_download(
+            'damo/nlp_gpt3_text-generation_chinese-base')
+        config: Config = read_config(
+            os.path.join(self.model_dir, 'configuration.json'))
+        config.megatron.world_size = 2
```

Comment:
No tp

Reply:
changed the relevant code,del tp in this file.